pokecli.py

```diff
@@ -182,10 +182,11 @@ def init_config():
     add_config(
         parser,
         load,
-        long_flag="--spin_forts",
+        long_flag="--spin",
```

Comment:
Yeah, out of scope. I've been teasing @douglascamata about this change for a few days. It was a proposal from a few days ago that solved some very specific problems that we just never seemed to implement: https://github.com/PokemonGoF/PokemonGo-Bot/issues/573#issuecomment-234757535